packages/components/global.d.ts

```diff
@@ -8,3 +8,7 @@ declare module '*.svg' {
   const content: React.FC<SvgProps>
   export default content
 }
+
+type SnackbarType = import('./src/components/Snackbar/Snackbar').SnackbarType
+// eslint-disable-next-line no-var
+declare var snackbar: SnackbarType
```

Comment:
Does `const` not work here?

Reply:
That is correct per [StackOverflow answer here](https://stackoverflow.com/a/56984941/13261302) and corresponding [TS documentation for global variables](https://devblogs.microsoft.com/typescript/announcing-typescript-3-4/#type-checking-for-globalthis):
> Note that global variables declared with `let` and `const` don’t show up on `globalThis`.

Admittedly didn't test if that had changed more recently.

Reply:
As a circle-back on this. I tried using `const` on my branch implementing it and it does work to launch the snackbar, but gives a TS error on `globalThis.snackbar` so seems the StackOverflow answer is correct that globalThis only recognizes `var`.